frontend/src/components/navigation/MobileNavigation.module.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.mobile-menu {
+  width: 100%;
+  position: absolute;
+  left: 0;
+  // setting the z-index low to allow mobile
+  // menu to slide behind main navigation header
+  z-index: 2;
+  top: $navHeight;
+  background: $color-white;
+  flex-direction: column;
+  box-shadow: $box-shadow-lg;
+
+  &:not(is-active) {
+    animation: slide-out 0.55s cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
+  &.is-active {
+    animation: slide-in 0.75s cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
```

Comment:
@Vinnl Do you think it would be enough with: Essentially, it would remove the animation. 

```
@media (prefers-reduced-motion) {
    &.not-active {
      animation: none;
    }
    &.is-active {
      animation: none;
    }
  }
```